src/ansi-c/scanner.l

```diff
@@ -899,8 +899,12 @@ enable_or_disable ("enable"|"disable")
 "__if_not_exists"   { return MSC_cpp_keyword(TOK_MSC_IF_NOT_EXISTS); }
 "__underlying_type" { return conditional_keyword(PARSER.cpp98, TOK_UNDERLYING_TYPE); }
 
-"[["                { if(PARSER.c23)
+"[["                { if(PARSER.c23 ||
+                         PARSER.mode==configt::ansi_ct::flavourt::GCC ||
```

Comment:
May I ask you to add a comment that gcc/clang support this C23 feature irrespectively of the standard set?

Reply:
Done (plus tested with more compilers and found that Visual Studio also started accepting those since 2022, also irrespective of the language standard selected on the command-line).